features/step_definitions/edit_restaurant.rb

```diff
@@ -0,0 +1,15 @@
+Given(/^the following restaurant exists$/) do |table|
+  table.hashes.each do |restaurant|
+    FactoryGirl.create(:restaurant, restaurant)
+  end
+end
+
```

Comment:
you can refactor the following step definitions into 1 since they do the same thing.
```gherkin
Given(/^I (?:am|should be) on the "([^"]*)" page for "([^"]*)"$/) do |page, name|
  restaurant = Restaurant.find_by(name: name)
  visit edit_restaurant_path(restaurant)
end
```

the group `(?:am|should be)` will match either of those expressions and because of `?:` it'll not be captured as input to the block like `page` and `name`